src/components/ContentNode/Aside.vue

```diff
@@ -10,7 +10,7 @@
 
 <template>
   <aside :class="kind" :aria-label="kind">
-    <p class="label">{{ label }}</p>
+    <p class="label">{{ name || $t(label) }}</p>
```

Comment:
Would `name` be localized as well?

Reply:
I believe so. This would be the case with a custom aside title provided by the markdown/json and not a hardcoded default.

Reply:
Exactly what Marcus pointed out